### PR TITLE
started migration sdk v2 -> v3

### PR DESCRIPTION
**Pull request by anovoselskyi**

### CODE REVIEW

File: ios/Nami.m [hunk 1]
```diff
@@ -66,20 +66,21 @@ @implementation NamiBridge (RCTExternModule)
...
-                config.developmentMode = [((NSNumber *)developmentModeString) boolValue];
-            } else if ([developmentModeString isKindOfClass:[NSString class]] ) {
-                if ([[((NSString *)developmentModeString) lowercaseString] hasPrefix:@"t"] )
-                {
-                    // bypass is false by default, so we only worry about checking for enabling bypass
-                    config.developmentMode = true;
-                }
-            }
-        }
+        //    Didn't found new API
```
Comment (namidan) [inline ios/Nami.m:69]: @anovoselskyi correct, can remove developmentMode reference

File: ios/NamiEntitlementManagerBridge.m [hunk 2]
```diff
@@ -64,48 +64,50 @@ @implementation NamiEntitlementManagerBridge (RCTExternModule)
...
-
-            [entitlementSetters addObject:setter];
-        } else {
-            NSLog(@"Warning, entitlement to set had empty referenceID, whole entitlement is \n %@", entitlementSetterDict);
-        }
-    }
-
-    [NamiEntitlementManager setEntitlements:entitlementSetters];
-}
+//    Didn't found new API
```
Comment (namidan) [inline ios/NamiEntitlementManagerBridge.m:67]: @anovoselskyi correct, this can be removed.

File: ios/Nami.m (hunk 1 above)
Comment (anovoselskyi) [reply ios/Nami.m:69]: sure

File: ios/NamiPaywallManagerBridge.m [hunk 3]
```diff
@@ -32,103 +32,106 @@ - (instancetype)init
...
+        }];        
     }
     return self;
 }
 
-RCT_EXTERN_METHOD(raisePaywall)
-- (void)raisePaywall {
-  [NamiPaywallManager raisePaywallFromVC:nil];
-}
+//  Didn't found new API
```
Comment (namidan) [inline ios/NamiPaywallManagerBridge.m:42]: @anovoselskyi correct, these are replaced by the new methods in NamiCampaignManager.launch

File: ios/NamiPaywallManagerBridge.m [hunk 4]
```diff
@@ -32,103 +32,106 @@ - (instancetype)init
...
+//RCT_EXTERN_METHOD(raisePaywall)
```
Comment (namidan) [inline ios/NamiPaywallManagerBridge.m:43]: @anovoselskyi these are replaced by the new methods in NamiCampaignManager.launch

File: ios/NamiPaywallManagerBridge.m [hunk 5]
```diff
@@ -32,103 +32,106 @@ - (instancetype)init
...
+//
+//
+//        NSArray *wrapperArray = @[@{ @"namiSkus": productDicts,
+//                                     @"developerPaywallID": developerPaywallID,
+//                                     @"paywallMetadata": paywallMeta }];
+//        completion(wrapperArray);
+//    }];
+//}
+
+//    Didn't found new API
```
Comment (namidan) [inline ios/NamiPaywallManagerBridge.m:106]: @anovoselskyi this one can be removed

File: ios/NamiPaywallManagerBridge.m [hunk 6]
```diff
@@ -32,103 +32,106 @@ - (instancetype)init
...
+//RCT_EXPORT_METHOD(paywallImpression:(NSString *)developerPaywallID)
+//{
+//    [NamiPaywallManager paywallImpressionWithDeveloperID:developerPaywallID];
+//}
+
+//    Didn't found new API
+//RCT_EXPORT_METHOD( preparePaywallForDisplay:(BOOL)backgroundImageRequired
```
Comment (namidan) [inline ios/NamiPaywallManagerBridge.m:113]: @anovoselskyi this one can be removed

File: ios/NamiPaywallManagerBridge.m [hunk 7]
```diff
@@ -32,103 +32,106 @@ - (instancetype)init
...
-RCT_EXPORT_METHOD(preparePaywallForDisplayByDeveloperPaywallId:(NSString *)developerPaywallID
-    backgroundImageRequired: (BOOL)backgroundImageRequired
-    imageFetchTimeout:(double)imageFetchTimeout )
-{
-    [NamiPaywallManager preparePaywallForDisplayWithDeveloperPaywallID:developerPaywallID backgroundImageRequired:backgroundImageRequired imageFetchTimeout:imageFetchTimeout prepareHandler:^(BOOL success, NSError * _Nullable error) {
-        [[NamiEmitter reactInstance] sendEventPreparePaywallForDisplayFinishedWithResult:success developerPaywallID:developerPaywallID error:error];
-    }];
-}
+//    Didn't found new API
+//RCT_EXPORT_METHOD(fetchCustomPaywallMetaForDeveloperID:(NSString *)developerPaywallID completion:(RCTResponseSenderBlock)completion)
```
Comment (namidan) [inline ios/NamiPaywallManagerBridge.m:78]: @anovoselskyi I need to check on this one

**Review by namidan — APPROVED** [review]

File: ios/NamiPaywallManagerBridge.m [hunk 8]
```diff
@@ -32,103 +32,78 @@ - (instancetype)init
...
-  [NamiPaywallManager raisePaywallFromVC:nil];
+    [NamiCampaignManager launch];
 }
 
-RCT_EXPORT_METHOD(raisePaywallByDeveloperPaywallId:(NSString * _Nonnull) developerPaywallID)
-{
-    [NamiPaywallManager raisePaywallWithDeveloperPaywallID:developerPaywallID fromVC:nil];
-}
+//    Didn't found new API
+//RCT_EXPORT_METHOD(raisePaywallByDeveloperPaywallId:(NSString * _Nonnull) developerPaywallID)
```
Comment (anovoselskyi) [inline ios/NamiPaywallManagerBridge.m:48]: Do we have a new API here?

File: ios/NamiPurchaseManagerBridge.m [hunk 9]
```diff
@@ -139,9 +139,10 @@ - (void) doSKUPurchaseWithSKUID:(nonnull NSString*)skuID namiPaywall:(NamiPaywal
 RCT_EXPORT_METHOD(buySKU:(nonnull NSString*)skuID paywallDeveloperID:(nonnull NSString*)paywallDeveloperID completion:(RCTResponseSenderBlock)completion)
 {
     if (paywallDeveloperID.length > 0) {
-        [NamiPaywallManager fetchCustomPaywallMetaForDeveloperID:paywallDeveloperID :^(NSArray<NamiSKU *> * _Nullable products, NSString * _Nonnull developerPaywallID, NamiPaywall * _Nullable namiPaywall)  {
-            [self doSKUPurchaseWithSKUID:skuID namiPaywall:namiPaywall completion:completion];
-        }];
+//        Didn't found new API
+//        [NamiPaywallManager fetchCustomPaywallMetaForDeveloperID:paywallDeveloperID :^(NSArray<NamiSKU *> * _Nullable products, NSString * _Nonnull developerPaywallID, NamiPaywall * _Nullable namiPaywall)  {
```
Comment (anovoselskyi) [inline ios/NamiPurchaseManagerBridge.m:143]: @burcaw do we have a new API here?

File: ios/NamiPaywallManagerBridge.m [hunk 10]
```diff
@@ -32,103 +32,78 @@ - (instancetype)init
...
 }
 
-RCT_EXPORT_METHOD(canRaisePaywall:(RCTResponseSenderBlock)completion)
-{
-    BOOL canRaise = [NamiPaywallManager canRaisePaywall];
-    completion(@[[NSNumber numberWithBool:canRaise]]);
-}
-
+//    Didn't found new API
+//RCT_EXPORT_METHOD(canRaisePaywall:(RCTResponseSenderBlock)completion)
```
Comment (anovoselskyi) [inline ios/NamiPaywallManagerBridge.m:59]: @burcaw do we have a new API here?

File: ios/NamiPaywallManagerBridge.m (hunk 7 above)
Comment (namidan) [reply ios/NamiPaywallManagerBridge.m:76]: @anovoselskyi go ahead and remove this one

File: ios/NamiPaywallManagerBridge.m (hunk 8 above)
Comment (namidan) [reply ios/NamiPaywallManagerBridge.m:48]: @anovoselskyi not in this class, this functionality will be served through NamiCampaignManager.launch

File: ios/NamiPurchaseManagerBridge.m (hunk 9 above)
Comment (namidan) [reply ios/NamiPurchaseManagerBridge.m:143]: @anovoselskyi remove

File: ios/NamiPaywallManagerBridge.m (hunk 10 above)
Comment (namidan) [reply ios/NamiPaywallManagerBridge.m:59]: @anovoselskyi remove

**Pull request merged**